build.gradle

```diff
@@ -3,9 +3,13 @@
 buildscript {
     repositories {
         jcenter()
+        maven {
+            url 'https://maven.google.com/'
+            name 'Google'
+        }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.0'
+        classpath 'com.android.tools.build:gradle:2.3.3'
```

Comment:
what's the reason for not updating to `3.0.1`?

Reply:
Good point! We'll update next time! 👍

Reply:
[here](https://github.com/gini/gini-sdk-android/pull/5) we go :)